packages/terra-clinical-data-grid/src/DataGrid.jsx

```diff
@@ -1103,5 +1103,7 @@ class DataGrid extends React.Component {
 
 DataGrid.propTypes = propTypes;
 DataGrid.defaultProps = defaultProps;
+DataGrid.Opts = {};
+DataGrid.Opts.ColumnSortIndicators = columnSortIndicators;
```

Comment:
Out of curiosity, why are we adding the Sort Indicator object to the DataGrid Opts? It doesn't look like we would need to use the sort indicator enum to render the DataGrid component?

Reply:
The sort indicator is used in the `columnDataShape` in the `pinnedColumns` and `overflowColumns` props which are on the DataGrid component.